openapi/v1/referenceGenome.yaml

```diff
@@ -8,6 +8,78 @@ tags:
   - name: Reference genome
 paths:
   /api/v1/reference-genomes:
+    get:
+      operationId: search_reference_genomes
+      summary: List or search for Reference Genome objects
+      tags:
+        - Reference genome
+      description: |-
+        Retrieve Reference Genome metadata objects by searching/listing Reference Genome metadata.
+
+        ## Metadata full-text queries
+        Single words can be supplied as is, otherwise use speech marks (`"`) to quote queries that include whitespace. Speech marks and backslash characters in the query need to be escaped with a backslash (`\`).
+
+        ## Paging
+        For performance reasons this endpoint returns results in "pages" of limited size. In order to retrieve all matching results the client needs to request multiple pages starting from an offset to the first result. You can do this using the pageOffset query parameter. A value of 0 (the default) instructs the server to return the first page of results, 100 would return a page of results starting from the 101st result and so on. To retrieve all results, iterate through pages by increasing the offset in multiples of limit (e.g., `offset = n * limit`), until all results have been retrieved. The total number of pages can be calculated by dividing the total number of results by the limit.
```

Comment:
- Comma after For performance reasons
 - Please grave the parameters (i.e. `pageOffset`)